packages/releaser/src/ReleaseWorker/CreateBranchReleaseWorker.php

```diff
@@ -14,7 +14,7 @@
      */
     public function getDescription(): string
     {
-        return '[Manual] Create branch "rc-beta-16"';
+        return '[Manual] Create branch "rc-<version-number>"';
```

Comment:
The `getDescription()` could actually accept `Version` object and here it should report the exact name. So the programmer doesn't have to think about the name "should it be 'rc-70', or 'rc-v70b' or 'rc-v70beta'?" 

```
[Manual] Create branch "rc-v7.0-beta"
```